src/assets/css/fonts.scss

```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: "fontHelvetica";
+  src: url("https://pushdev-code.github.io/shoping-cart/public/fonts/helveticaneue_medium-webfont.woff2");
+  font-size: 14px;
+  line-height: 0.9;
+  text-transform: uppercase;
```

Comment:
keep these properties in the elements that use this font-face.

Reply:
Done